src/dom.ts

```diff
@@ -51,10 +58,8 @@ export function fromTreeProtoHelper(
       continue;
     }
 
-    // ZERO_LENGTH nodes get flattened into the parent.
     if (node.tagid === getTagId('ZERO_LENGTH')) {
-      fromTreeProtoHelper(node.children, doc, parent);
-      continue;
+      throw new Error(`Found a #document in a non-root position`);
```

Comment:
Why is `getTagId` used with literals? I can see it being useful for variables but with literals I feel like we should just use the enum.

Reply:
Only in that I wasn't planning on exporting the map. I can though!